mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +471,45 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _events_from_eeglab_raw(fname, event_id=dict()):
+    """Create events array from EEGLAB structure by looking them up in the
+    event_id, trying to reduce them to their integer part otherwise, and
+    entirely dropping them (with a warning) if this is impossible"""
+    from scipy.io import loadmat
+
+    eeg = loadmat(fname, struct_as_record=False, squeeze_me=True)["EEG"]
```

Comment:
Ah, because it used to be a standalone function.
I'll pass `eeg` as the 1st argument instead.